pom.xml

```diff
@@ -330,13 +333,15 @@
                             <exclude>**/spring.factories</exclude>
                             <exclude>**/spring.provides</exclude>
                             <exclude>**/*.iml</exclude>
+                            <exclude>**/nb-configuration.xml</exclude>
+                            <exclude>**/faces-config.NavData</exclude>
```

Comment:
This could also be in the `.gitignore` file.

Reply:
removed